component/libfuse/libfuse.go

```diff
@@ -188,6 +188,8 @@ func (lf *Libfuse) Validate(opt *LibfuseOptions) error {
 	lf.ignoreOpenFlags = opt.IgnoreOpenFlags
 	lf.nonEmptyMount = opt.nonEmptyMount
 	lf.directIO = opt.DirectIO
+	lf.ownerGID = opt.Gid
+	lf.ownerUID = opt.Uid
```

Comment:
Can we add some tests for this fix?

Reply:
We do not have libfuse level UT as this requires full mount and all. But I agree we need to think of some way to at least come up with some config level UT.

Reply:
E2E or some other mount test. We can think about it and have this some other PR.